src/main.rs

```diff
@@ -240,7 +240,7 @@ fn run() -> Result<()> {
         runner.execute(Step::Shell, "zinit", || zsh::run_zinit(&base_dirs, run_type))?;
         runner.execute(Step::Shell, "oh-my-zsh", || zsh::run_oh_my_zsh(&ctx))?;
         runner.execute(Step::Shell, "fisher", || unix::run_fisher(&base_dirs, run_type))?;
-        runner.execute(Step::Shell, "oh-my-fish", || unix::run_oh_my_fish(&ctx))?;
+        runner.execute(Step::Shell, "oh-my-fish", || unix::run_oh_my_fish(&base_dirs, run_type))?;
```

Comment:
Passing `&ctx` is the preferred way now. Can you please do that?

Reply:
So, could you please tell me how to pass `&base_dirs` with `&ctx`? I'm unfamiliar with Rust...

Or maybe `run_oh_my_fish(&base_dirs, &ctx)`?

Reply:
`ctx.base_dirs()` to access the base directories

Reply:
Done.